frontend/src/components/EmployeeDetails.js

```diff
@@ -134,6 +150,13 @@ const EmployeeDetails = ({employee}) => {
             <p><strong>Numer konta: </strong>{employee.numer_konta}</p>
             <p><strong>Adres: </strong>{employee.adres}</p>
             <p><strong>ID Kliniki: </strong>{employee.id_kliniki}</p>
+            <p style={{color: "#E5BA73"}} onClick={() => setShowed(showed => !showed)}><strong>Pokaż szczegóły kliniki:</strong></p>
+            {showed ? 
+            <div id="patient-details">
+                {clinics && clinics.filter(p => (p._id === employee.id_kliniki)).map(p => (
```

Comment:
We are dealing with employees, so it should be e?

Reply:
We are dealing with clinics here, changed to 'c'